src/views/components/input/rich-text-editor/RichTextEditor.web.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import { Editor } from 'slate-react';
 import { Value } from 'slate';
+import Html from 'slate-html-serializer';
 import { isKeyHotkey } from 'is-hotkey';
-import { string, oneOfType, number } from 'prop-types';
+import { string, oneOfType, number, func } from 'prop-types';
 /* This could be dynamic in the future (pre filled schemas while typing in the job description.) */
 import initialValue from './defaultSchema.json';
```

Comment:
initial value should be taken from value prop. use componentDidMount and componentDidUpdate to check for a new value prop and call a function to compute and set state.value accordingly.

Reply:
looks good